app/Http/Controllers/QuizController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\QuestionResource;
+use App\Models\Question;
+use App\UseCases\Quiz\DestroyUseCase;
+use App\UseCases\Quiz\StoreUseCase;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use App\Http\Requests\Quiz\StoreRequest;
+use App\UseCases\Quiz\UpdateUseCase;
+use App\Http\Requests\Quiz\UpdateRequest;
+
+class QuizController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param int $material_id
+     * @return AnonymousResourceCollection
+     */
+    public function index(int $material_id)
+    {
+        // 教材に紐付いているクイズの取得
+        return QuestionResource::collection(Question::where('material_id', $material_id)->get());
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param StoreRequest $request
+     * @param StoreUseCase $useCase
+     * @return QuestionResource
+     */
+    public function store(StoreRequest $request, StoreUseCase $useCase)
```

Comment:
docコメントで帰り値が分かっているくらいであれば、もうひと労力で関数の帰り値に型ヒントを書いておくと良さそうです。
```suggestion
    public function store(StoreRequest $request, StoreUseCase $useCase): QuestionResource
```
型が違うものが帰ろうとした際に実行時エラーになってくれます
他のメソッドについても同様です
他の作業が一息ついて余裕ができたら別PRで修正できると良さそうです。